HOWTORELEASE.rst

```diff
@@ -3,90 +3,83 @@ How to release pytest
 
 Note: this assumes you have already registered on pypi.
 
-0. create the branch release-VERSION
-   use features as base for minor/major releases
-   and master as base for bugfix releases
+1. Bump version numbers in ``_pytest/__init__.py`` (``setup.py`` reads it).
 
-1. Bump version numbers in _pytest/__init__.py (setup.py reads it)
+2. Check and finalize ``CHANGELOG.rst``.
 
-2. Check and finalize CHANGELOG
+3. Write ``doc/en/announce/release-VERSION.txt`` and include
+   it in ``doc/en/announce/index.txt``. Run this command to list names of authors involved::
 
-3. Write doc/en/announce/release-VERSION.txt and include
-   it in doc/en/announce/index.txt::
+        git log 2.8.2..HEAD --format='%aN' | sort -u
```

Comment:
`git log $(git tag --sort=version:refname | tail -n1)..HEAD --format='%aN' | sort -u` to make it use the last tag automatically?